### PR TITLE
Disable the Dependency Check step until GA

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
@benbp if there's a better way to disable this step please let me know.

**Comment by benbp** [comment]
@jhendrixMSFT this will work fine as a temporary disable method. FYI @tadelesh @seankane-msft.

File: eng/pipelines/templates/steps/analyze.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ steps:
 
   - task: Powershell@2
     displayName: 'Dependency Check'
+    condition: false
```
Comment (weshaggard) [inline eng/pipelines/templates/steps/analyze.yml:13]:
```suggestion
    continueOnError: true
```

You could also just continue on error so the step runs but doesn't fail the build.

Comment (jhendrixMSFT) [reply eng/pipelines/templates/steps/analyze.yml]: Good idea, fixed.

**Review by benbp — APPROVED** [review]

**Pull request merged**